doc/api/fs.md

```diff
@@ -199,8 +199,7 @@ changes:
 * `data` {string|Buffer|TypedArray|DataView|AsyncIterable|Iterable|Stream}
 * `options` {Object|string}
   * `encoding` {string|null} **Default:** `'utf8'`
-  * `flush` {boolean} If `true`, the underlying file descriptor is flushed
```

Comment:
I think that the `flush` option should not be removed.

Reply:
I deleted for the following reason. What do you think?

- The code seems not to be using the flush option, as shown in the code below.
https://github.com/nodejs/node/blob/1781f6363359f3d0d944360c13f9e6206938418d/lib/internal/fs/promises.js#L1203-L1204
- It is not explicitly stated in the [documentation](https://nodejs.org/docs/latest-v23.x/api/fs.html#filehandlewritefiledata-options) for `filehandle.writeFile`, which is an alias for this method.

Reply:
The change this time is about `fileHandle`, and the commit you showed doesn't seem to have test cases for `fileHandle`.

Reply:
You are right.